packages/flutter/lib/src/material/checkbox_list_tile.dart

```diff
@@ -518,10 +518,13 @@ class CheckboxListTile extends StatelessWidget {
         );
     }
 
-    final (Widget? leading, Widget? trailing) = switch (controlAffinity) {
+    final ListTileThemeData listTileTheme = ListTileTheme.of(context);
+    final ListTileControlAffinity effectiveControlAffinity = controlAffinity ?? listTileTheme.controlAffinity ?? ListTileControlAffinity.platform;
```

Comment:
```suggestion
    final ListTileControlAffinity effectiveControlAffinity = controlAffinity ??
      listTileTheme.controlAffinity ?? ListTileControlAffinity.platform;
```

`effectiveControlAffinity` are bit long for single. we can split them in all classes

Reply:
I reformatted the code